### PR TITLE
Fix invalid default cfg value, add netlify support

### PR DESCRIPTION
**Pull request by jvyden**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (Hughp135) [inline install.sh]: i feel like this file is kinda unnecessary.. if a user can't type two commands then there's some other issues going on there :joy:

Comment (jvyden420) [reply install.sh]: meh, i like having the option to just run a script and walk away :stuck_out_tongue: 
ill move this file to the gist i guess

Comment (Hughp135) [reply install.sh]: well I agree it's convenient to have a script that runs this all for you, however, every npm package should be a case of just `npm i` and `npm start`. sure there are 2 folders you have to run this in but it's not exactly rocket science. Having an install script like this adds another file to maintain, which may get outdated and cause confusion.

File: front-end/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export const environment = {
   production: true,
-  base_api_url: '/api/',
+  base_api_url: 'https://play.osustuff.org/api/',
```
Comment (jvyden420) [inline front-end/src/environments/environment.prod.ts]: Blegh. I *still* don't like the way this works, months later. Is there a better way I can do this?

File: front-end/src/_redirects [hunk 3]
```diff
@@ -0,0 +1 @@
+/*    /index.html    200
```
Comment (jvyden420) [inline front-end/src/_redirects]: I think this was to get netlify to redirect everything through the index so the angular router worked properly.

File: README.md [hunk 4]
```diff
@@ -65,3 +65,7 @@ window.adminLogon('asd123') // (default admin password)
 
 
 After entering this javascript command, you can refresh the page to see the 'admin' button on the lobbies page. This allows you to run testing commands such as clearing the database, toggling lobby auto-creation, and skipping the current round.
+
+### 5) Commit hook (optional)
+
+If you'd like a quick hook to make sure your lint passes before you commit code, head [here](https://gist.github.com/jvyden420/ef5962caa87fbe36c597ddd0018be501)
```
Comment (jvyden420) [inline README.md]: ez plug

File: back-end/config/default.yml [hunk 5]
```diff
@@ -10,7 +10,7 @@ ADMIN_PASS: 'asd123'
 OSU_OAUTH_ID: null
 OSU_OAUTH_SECRET: ''
 JWT_SECRET: 'secret'
-OSU_OAUTH_REDIRECT: 'http://localhost:4200/api/login-redirect'
+OSU_OAUTH_REDIRECT: 'http://localhost:4200/api/login-verify'
```
Comment (jvyden420) [inline back-end/config/default.yml]: Actually, I may have been the one to break this in an earlier commit. Oops.

**Comment by jvyden420** [comment]
now *my* clock is broken. 
image

**Pull request closed without merging**